modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -988,6 +1134,9 @@ FeatureFloodCount::FeatureData::merge(FeatureData && rhs)
   else
     std::move(rhs._bboxes.begin(), rhs._bboxes.end(), std::back_inserter(_bboxes));
 
+  // Keep track of the original ids so we can notify other processors of the local to global mapping
+  _orig_ids.splice(_orig_ids.end(), std::move(rhs._orig_ids));
```

Comment:
It's a great data structure if you write to it a lot but don't really care about ordering/sorting. maps and sets "feel" easy to use but consume a lot of disjoint memory and they make a lot of trips to the allocator during use and rebalancing. std::list is used here in a few places because it's exactly the right data structure.